src/Space.ts

```diff
@@ -53,18 +55,28 @@ class Space extends EventEmitter<SpaceEventsMap> {
     this.locations = new Locations(this, this.presenceUpdate);
     this.cursors = new Cursors(this);
     this.members = new Members(this);
+    this.locks = new Locks(this, this.presenceUpdate);
   }
 
-  private presenceUpdate = (data: PresenceMember['data']) => {
-    return this.channel.presence.update(data);
+  private presenceUpdate = (data: PresenceMember['data'], extras?: any) => {
```

Comment:
it would be useful to add a type to `extras`:

```
extras?: {
  locks: LockRequest[];
}
```

Reply:
Done in ba509cc.